apps/src/p5lab/spritelab/commands/actionCommands.js

```diff
@@ -169,6 +179,32 @@ export const commands = {
     return touching;
   },
 
+  isDirectlyAbove(spriteArg, targetArg) {
+    let sprites = this.getSpriteArray(spriteArg);
+    let targets = this.getSpriteArray(targetArg);
+    let touching = false;
+    sprites.forEach(sprite => {
+      const spriteCollider = createSpriteCollider(sprite);
+      if (spriteCollider.bottom >= APP_HEIGHT) {
+        touching = true;
+      } else {
+        for (const target of targets) {
+          const targetCollider = createSpriteCollider(target);
+
+          if (
+            spriteCollider.bottom === targetCollider.top &&
```

Comment:
Are these numbers pixel locations? Should we have some sort of buffer (ex top is between `bottom - 2` and `bottom + 2`)? This might just be something we figure out by using the blocks so it could be a follow up.

Reply:
Yes, they're essentially pixel locations. For now, it's working as expected, but I think we could certainly add a buffer if/when needed.